examples/with-react-native-web/packages/ui/src/button.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import {
+  Pressable,
+  StyleSheet,
+  GestureResponderEvent,
+  Text,
+} from "react-native";
+
+export interface ButtonProps {
+  text: string;
+  onClick?: ((event: GestureResponderEvent) => void) | null | undefined;
+}
+
+export function Button({ text, onClick }: ButtonProps) {
+  return (
+    <Pressable style={styles.button} onPress={onClick}>
```

Comment:
**nitpick:** Consider using simpler alternatives‏

The `Pressable` component is meant to be used in conjunction with either a functional `style` prop or functional `children` prop to apply different behaviours based on the interaction state, which we're not doing here - example [here](https://reactnative.dev/docs/pressable#example). It would be easier to use `TouchableOpactity` instead.

Reply:
Awesome, thanks for this context - swapped it to `TouchableOpacity`!